site/content/en/docs/Installation/_index.md

```diff
@@ -46,48 +46,82 @@ gcloud container clusters create --machine-type n1-standard-2 open-match-dev-clu
 
 [Set up a Local Minikube cluster](https://kubernetes.io/docs/setup/minikube/)
 
 ## Install Core Open Match
 
-Open Match comprises of a set of services that enable core functionality such as Ticket management, filtering, Match generation etc. It also includes a
-state storage that Open Match uses to persist state needed for its functioning.
+Open Match comprises of a set of services that enable core functionality such as Ticket Management and Filtering etc. It also includes a
+state storage to persist state needed for its functioning.
 
 The simplest way to install Open Match is to use the install.yaml files for the latest release.
 This installs Open Match with the default configuration.
 
 ```bash
 # Create a cluster role binding (if using gcloud on Linux or OSX)
-kubectl create clusterrolebinding cluster-admin-binding --clusterrole cluster-admin --user `gcloud config get-value account`
+> kubectl create clusterrolebinding cluster-admin-binding --clusterrole cluster-admin --user `gcloud config get-value account`
 
 # Create a cluster role binding (if using gcloud on Windows)
-for /F %i in ('gcloud config get-value account') do kubectl create clusterrolebinding cluster-admin-binding --clusterrole cluster-admin --user %i
+> for /F %i in ('gcloud config get-value account') do kubectl create clusterrolebinding cluster-admin-binding --clusterrole cluster-admin --user %i
 
 # Create a cluster role binding (if using minikube)
-kubectl create clusterrolebinding cluster-admin-binding --clusterrole=cluster-admin --serviceaccount=kube-system:default
-
-# Create a namespace to place all the Open Match components in.
-kubectl create namespace open-match
+> kubectl create clusterrolebinding cluster-admin-binding --clusterrole=cluster-admin --serviceaccount=kube-system:default
 
 # Install the core Open Match services and monitoring services.
-kubectl apply -f https://open-match.dev/install/v{{< param release_version >}}/yaml/install.yaml --namespace open-match
+> kubectl apply -f https://open-match.dev/install/v{{< param release_version >}}/yaml/install.yaml --namespace open-match
 ```
 
+You should be able to see the output below from your console. The `01-open-match-core.yaml` file contains:
 
-## Install Components and Configure Open Match
-
-In order to run Open Match, it must be configured by:
+  - A Redis deployment as Open Match's state storage system.
+  - ServiceAccounts, Roles and RoleBindings to define Open Match deployments' IAMs. 
+  - Several Services for Open Match endpoints. 
+  - HorizontalAutoScalars to auto-scale Open Match based on pods' average CPU utilization. 
 
-- Installing a match making function and evaluator.
-- Updating the configmap with the correct indexes.
-
-[Getting Started]({{< ref "/docs/Getting Started" >}}) contains an example which covers this.
+```
```

Comment:
What's this section for?  This is likely to change a lot and not be of much use?

Reply:
I added this section to address Mark's concern. He pointed out that

>  Wow this is a lot of things installed. Would be good to have a component list somewhere.
Why do I have cassandra in here?

I truncated this list a bit to make it less redundant.

Reply:
I think the section is reasonable, but the actual list of things created isn't.  Lots of information with no context or explanation is just noise.

Reply:
Deleted.